### PR TITLE
New update

### PR DESCRIPTION
**Pull request by jhaastrup**
- Updated the home page
-  Updated the authentication page 
- Updated the queries and mutation page

### CODE REVIEW

File: docs/authentication.mdx [hunk 1]
```diff
@@ -8,12 +8,21 @@ sidebar_label: "Authentication"
...
 import CodeBlock from '@theme/CodeBlock';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
 
 
 Authenticate your API calls by including your API key in the Authorization header of every request you make. 
 You can manage your API keys from the [dashboard](https://dashboard.usedropper.com/).
 
 
+## How do i get my API key?
```
Comment (kourtc) [inline docs/authentication.mdx:19]: Typo

File: docs/mutations/create-a-delivery.md [hunk 2]
```diff
@@ -2,9 +2,15 @@
 sidebar_position: 1
 ---
 
-# Create Deliveries
+# Create Delivery
 
-Let's write a simple query to create a new delivery.
+Create delivery mutation is used to request a new delivery package for your user. 
+to do this, you make a mutation request to the newDelivery endpoint by passing required parameters in the request body like ```sender_name``` ```sender_address``` ```recipent_name``` ```recipent_address``` and passing your API key in the request header.
```
Comment (kourtc) [inline docs/mutations/create-a-delivery.md]: newDelivery is not an endpoint. can this be reworded?

File: docs/mutations/create-a-delivery.md [hunk 3]
```diff
@@ -2,9 +2,15 @@
...
+
+Think about mutations like a post request in REST.
```
Comment (kourtc) [inline docs/mutations/create-a-delivery.md:10]: Nice. This is helpful for graphql beginners.

File: docs/queries/get-delivery-status.md [hunk 4]
```diff
@@ -3,9 +3,16 @@ sidebar_position: 4
 ---
 
 # Get Delivery Status
-You can fetch a delivery with a given `tracking_id`.
+  
+Get Delivery query is used to get all detials of a particular delivery.
```
Comment (kourtc) [inline docs/queries/get-delivery-status.md:7]: typo in details

File: src/pages/index.mdx [hunk 5]
```diff
@@ -24,6 +24,8 @@ export const Tagline = () => <p style={{'font-size': '1.5em'}} >{useDocusaurusCo
 
 <Tagline/>
 
+#### Dropper is building an ultra reliable order fulfillment internet infrastructure for business in Africa.
```
Comment (kourtc) [inline src/pages/index.mdx:27]: Please remove this.

**Comment by jhaastrup** [comment]
okay will update it

**Pull request merged**